lib/feature_flipper.rb

```diff
@@ -6,7 +6,7 @@ def self.send_email?
   end
 
   def self.staging_email?
-    Settings.reports.server.include?('stage')
+    (Settings.hostname =~ /-api/).present?
```

Comment:
Can we just set this to the value that it should be in settings.local.yml instead of checking hostnames?

Reply:
Sure.  What I'm really looking for here is a flag to tell if we're in production or not. That'll probably be useful in more than one situation.
I thought using the hostname would be a good way to do that, but this is susceptible to changes to the host subdomain.
To be even more future-proof, what if we add an entry to settings.local.yml that just lists the name of the deployed environment?

Reply:
FWIW this has been brought up before - https://github.com/department-of-veterans-affairs/vets-api/pull/2664

Reply:
+1 to implement #2664

Reply:
Since that conversation has already been had, I'm going to:
1. Close this
2. Open a devops PR
3. Open a new API PR.